charon/tests/ui/issue-297-cfg.rs

```diff
@@ -0,0 +1,31 @@
+fn f1(a: &[u8]) -> usize {
+    let mut sampled = 0;
+    if a[0] < 42 && a[1] < 16 {
+        sampled += 100;
+    }
+    sampled += 101;
+    sampled
+}
+
+const FIELD_MODULUS: i16 = 8;
+fn f2(a: &[u8], result: &mut [i16]) -> usize {
+    let mut sampled = 0;
+    for bytes in a.chunks(3) {
```

Comment:
The downside of `for` loops is that they make us import the whole of `Iterator`. For future reference in tests I'd prefer we use `while` loops.